### ISSUE TITLE
Insufficient color contrast for focus indicators [QAC]

### ISSUE THREAD

**Issue by ftourangeau** [MeD-DMC/Adapt-accessibilityfixes#198]
**COURSE:**
https://esdc-edsc-adapt.canadacentral.cloudapp.azure.com/preview/5dc446fd27f08836496b6b9e/63a32079b75e454dafed1e1d/
*******************************************************************************
Page:
Main Hamburger Menu
Lesson 1: Overview of electronic Passport Next Generation
Lesson 2: Domestic Print
Lesson 4: Manual QA

Description:
The focus indicator in the Drawer dialog, e.g.:
   • "Glossary..." link
has insufficient color contrast. The element was calculated to have the following traits
   • Foreground: `#CD1C6A`
   • Background: `#1C4E8A`
   • The contrast ratio is: 2.8:1

Any visual information necessary to indicate a state, such as whether a component is selected or focused must also ensure that the information used to identify the control in that state has a minimum 3:1 contrast ratio.
*******************************************************************************
Page:
